### ISSUE TITLE
[MapCard] Testing

### ISSUE THREAD

**Issue by davidicus** [carbon-design-system/carbon-addons-iot-react#2269]
Component needs comprehensive testing in a few areas. This issue is considered complete when:

- [ ] data test ids have been provided to appropriate pieces 
- [ ] Unit test are 80%+
- [ ] a11y scans in storybook do not show any violations 
- [ ] a11y unit test has been written using the IBM scan tool
- [ ] RTL has been tested and working properly. 
- [ ] Confirm that all strings are able to be substituted for i18n
